.github/workflows/test_tenant_ui.yml

```diff
@@ -0,0 +1,57 @@
+# This workflow will do a clean installation of node dependencies, cache/restore them, build the source code and run tests across different versions of node
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-nodejs
+
+name: Build and Test Tenant UI
+
+on:
+  push:
+    branches: ["develop"]
+  pull_request:
+    branches: ["develop"]
+
+jobs:
+  backend:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./services/tenant-ui
+
+    strategy:
+      matrix:
+        node-version: [16.x, 18.x, 20.x]
```

Comment:
Unless we still depend on `16.x` for some reason I would remove it, it is EOL anyway in September of 2023.